README.md

```diff
@@ -0,0 +1,72 @@
+# Visualization API
+
+This repository contains app that fetch (serverless) **BIRAFFE2** dataset and then visualizate it.
+
+The **"BIRAFFE2: The 2nd Study in Bio-Reactions and Faces for Emotion-based Personalization for AI
+Systems"** dataset is part of the work carried out by the AfCAI Research Group in Kraków, Poland.
+
+**BIRAFFE2** source: https://zenodo.org/record/3865860#.XvjpwecwhPY
+
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [Getting started](#getting-started)
+- [Scripts](#scripts)
```

Comment:
Replace `Scripts` field with `Datasets`.

Add a section about Continuous Integration.

Reply:
In section about CI touch about
 - test stage and each step
 - deploy step

Reply:
Add a section about `API`. It should include information like:

```
*TODO:*  implement API specs

Visualization API will output data from dataset in format:

```ts
{
dataset: unknown;
supportedVisualizations: Visualization[];
}
```

As of now `supportedVisualization` will be an empty array.